python/python/glide/async_commands/sorted_set.py

```diff
@@ -23,6 +23,23 @@ class InfBound(Enum):
     """
 
 
+class ScoreModifier(Enum):
```

Comment:
Rename to `ScoreFilter` https://github.com/aws/glide-for-redis/pull/1390#discussion_r1597277437